src/node.cc

```diff
@@ -779,134 +779,6 @@ Local<Value> MakeCallback(Isolate* isolate,
-  // truncate off the first 62 characters, but it caused numerous other
-  // problems when vm.runIn*Context() methods were used for non-module
-  // code.
-  //
-  // If we ever decide to re-instate such a hack, the following steps
-  // must be taken:
-  //
-  // 1. Pass a flag around to say "this code was wrapped"
-  // 2. Update the stack frame output so that it is also correct.
-  //
-  // It would probably be simpler to add a line rather than add some
-  // number of characters to the first line, since V8 truncates the
-  // sourceline to 78 characters, and we end up not providing very much
-  // useful debugging info to the user if we remove 62 characters.
-
-  int script_start =
-      (linenum - origin.ResourceLineOffset()->Value()) == 1 ?
-          origin.ResourceColumnOffset()->Value() : 0;
-  int start = message->GetStartColumn(env->context()).FromMaybe(0);
-  int end = message->GetEndColumn(env->context()).FromMaybe(0);
-  if (start >= script_start) {
-    CHECK_GE(end, start);
-    start -= script_start;
-    end -= script_start;
-  }
-
-  char arrow[1024];
-  int max_off = sizeof(arrow) - 2;
-
-  int off = snprintf(arrow,
-                     sizeof(arrow),
-                     "%s:%i\n%s\n",
-                     filename_string,
-                     linenum,
-                     sourceline_string);
-  CHECK_GE(off, 0);
-  if (off > max_off) {
-    off = max_off;
-  }
-
-  // Print wavy underline (GetUnderline is deprecated).
-  for (int i = 0; i < start; i++) {
-    if (sourceline_string[i] == '\0' || off >= max_off) {
-      break;
-    }
-    CHECK_LT(off, max_off);
-    arrow[off++] = (sourceline_string[i] == '\t') ? '\t' : ' ';
-  }
-  for (int i = start; i < end; i++) {
-    if (sourceline_string[i] == '\0' || off >= max_off) {
-      break;
-    }
-    CHECK_LT(off, max_off);
-    arrow[off++] = '^';
-  }
-  CHECK_LE(off, max_off);
-  arrow[off] = '\n';
-  arrow[off + 1] = '\0';
-
-  Local<String> arrow_str = String::NewFromUtf8(env->isolate(), arrow,
-      NewStringType::kNormal).ToLocalChecked();
-
-  const bool can_set_arrow = !arrow_str.IsEmpty() && !err_obj.IsEmpty();
-  // If allocating arrow_str failed, print it out. There's not much else to do.
-  // If it's not an error, but something needs to be printed out because
-  // it's a fatal exception, also print it out from here.
-  // Otherwise, the arrow property will be attached to the object and handled
-  // by the caller.
-  if (!can_set_arrow || (mode == FATAL_ERROR && !err_obj->IsNativeError())) {
-    if (env->printed_error())
-      return;
-    Mutex::ScopedLock lock(process_mutex);
-    env->set_printed_error(true);
-
-    uv_tty_reset_mode();
-    PrintErrorString("\n%s", arrow);
-    return;
-  }
-
-  CHECK(err_obj->SetPrivate(
-            env->context(),
-            env->arrow_message_private_symbol(),
-            arrow_str).FromMaybe(false));
-}
-
 
 void ReportException(Environment* env,
```

Comment:
From a quick check it seems like this function could move to `node_errors.h`. IMHO that makes sense.